thirdparty/fetch-thirdparty-deps.sh

```diff
@@ -39,10 +39,13 @@ fi
 
 if [ ! -f SharpFont.dll ]; then
 	echo "Fetching SharpFont from nuget"
-	nuget install SharpFont -Version 2.5.0.1
-	cp ./SharpFont.2.5.0.1/lib/net20/SharpFont* .
-	sed '/osx/s@\(dll="\)[^"]*\(" />\)@\1/Library/Frameworks/Mono.framework/Libraries/libfreetype.6.dylib\2@' ./SharpFont.2.5.0.1/Content/SharpFont.dll.config > SharpFont.dll.config
-	rm -rf SharpFont.2.5.0.1
+	nuget install SharpFont -Version 3.0.1
```

Comment:
You can then add ` -ExcludeVersion` version here as we learned in https://github.com/OpenRA/OpenRA/pull/7907.